.github/actions/spelling/candidate.patterns

```diff
@@ -161,6 +179,12 @@ GHSA(?:-[0-9a-z]{4}){3}
 # GitLab commits
 \bgitlab\.[^/\s"]*/(?:[^/\s"]+/){2}commits?/[0-9a-f]+\b
 
+# #includes
+^\s*#include\s*(?:<.*?>|".*?")
```

Comment:
Hmm, it would be nice if the patterns could be limited to specific file paths, so `#includes` only for `*.h/*.c/*.hpp/*.cpp/*.cc`. Same with `uses:` only in `.github/**/*.yaml`.

Reply:
Yeah... It's a thing that comes up _occasionally_, but then I'd have to make the file format *much* more complicated.